module-sys/SystemWatchdog/include/SystemWatchdog/SystemWatchdog.hpp

```diff
@@ -36,6 +36,9 @@ namespace sys
 
         void refresh() override;
 
+        // It doesn't disable watchdog, but it simply close the petting thread
```

Comment:
What does 'petting' mean in this context?

Reply:
watchdog petting - make sure it do not trigger